scripts/db_file_count.py

```diff
@@ -41,8 +41,12 @@
 )
 
 bucket = s3.Bucket(os.getenv('STORAGE_LOCATION_BUCKET'))
-prefix = os.getenv('STORAGE_LOCATION_KEY') + "/" + year + "/" + month + "/" + datetime_str + "/"
-
+key = os.getenv('STORAGE_LOCATION_KEY')
+if(key):
```

Comment:
just a personal preference

```suggestion
if key:
```